.github/workflows/build-website.yml

```diff
@@ -14,8 +14,26 @@ on:
       - ".github/workflows/*-website.yml"
 
 jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: "16"
+
+      - name: Run prettier
+        run: |
+          cd website
+          npm install
```

Comment:
Do we need to install everything just to run fmt?
I know it's kind of annoying to manually install stuff but still, it might reduce the runtime significantly

Reply:
Haven't tested. It might be possible to run `npx prettier`. I'll test that when I get a chance

Reply:
Well, I removed that entirely. Everything is in one job now